v21.1/backup.md

```diff
@@ -253,7 +253,7 @@ AS OF SYSTEM TIME '-10s'
 WITH DETACHED;
 ~~~
 
-The job ID is returned immediately without waiting for the job to finish:
+The job ID is returned after the backup is validated:
```

Comment:
I think it might good to link "after backup job creation" in the examples to that same architecture section. Since the reader could come to the examples before the options table, and therefore might not understand what "job creation" means.